src/coreComponents/common/Units.cpp

```diff
@@ -37,6 +37,11 @@ TimeFormatInfo::TimeFormatInfo( double const totalSeconds, int const years, int
 string TimeFormatInfo::toString() const
 {
   std::ostringstream oss;
+  constexpr double negativeTime = -1.0 + LvArray::NumericLimits< double >::epsilon;
+  if( m_totalSeconds < negativeTime )
+  {
+    oss << "-(";
+  }
```

Comment:
Initially I had `if( m_totalSeconds < 0.0 )` which was printing something like
```
-(00h00m00s) (-1.11e-07 s)
```
I didn't like the negative sign here. But I agree that it might be it is good to highlight that the time is negative.